test/test_torch.py

```diff
@@ -2738,6 +2739,7 @@ def test_qtensor(self):
         self.assertEqual(qr.q_zero_point(), zero_point)
         self.assertTrue(qr.is_quantized)
         self.assertFalse(r.is_quantized)
+        self.assertEqual(qr.qscheme(), 1) #torch.per_tensor_affine)
```

Comment:
@gchanan how can I have `qr.qscheme()` return `torch.per_tensor_affine` rather than `1` here?

Reply:
see below, the problem is in wrap_outputs.